.golangci.yaml

```
@@ -28,6 +28,9 @@ linters:
       - linters:
           - staticcheck
         text: "QF1006:" # could lift into loop condition (staticcheck)
+      - linters:
+          - staticcheck
+        text: "SA1019:" # aws-sdk-go v1 deprecated warning
```

Instead of suppressing all deprecation warnings, how about a message specific to aws?
```suggestion
        text: "Use aws-sdk-go-v2" # aws-sdk-go v1 deprecated warning
```

---

done